src/main/scala/mesosphere/marathon/MarathonConf.scala

```diff
@@ -224,4 +225,13 @@ trait MarathonConf
     descr = "The timeout for preparing the Marathon instance when elected as leader.",
     default = Some(3 * 60 * 1000L) //3 minutes
   )
+
+  lazy val maxQueuedRootGroupUpdates = opt[Int]("max_queued_root_group_updates",
```

Comment:
Kind of like the idea to have the var named internalXX, but the parameter not.
If we think we should unhide this parameter, we can make this change without breaking any existing external usage.